.github/workflows/release.yml

```diff
@@ -124,7 +124,9 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           body: ${{ steps.create_changelog.outputs.content }}
           tag_name: ${{ steps.create_changelog.outputs.version }}
-          files: pglt_*
+          files: |
+            pglt_*
+            docs/schemas/latest/schema.json
```

Comment:
Nice! Should we also regenerate the schema as part of the release workflow?

Reply:
we already check that the schema is generated as part of the pull request workflow, so it should always be up to date if changes are made via pull requests